website/src/menu.tsx

```diff
@@ -100,11 +108,18 @@ const SubMenu = ({ item, location }: { location: Location; item: any }) => {
   )
 }
 
-export const MobileNav = () => {
+export const MobileNav = (p: { menuID: number; cwkw?: boolean }) => {
```

Comment:
Yeah so get rid of this `cwkw` prop and make all necessary changes through the theme.